### PR TITLE
Completed typescript-operators-and-expressions exercise

### PR DESCRIPTION
**Pull request by BlakexLim**
typescript ops

### CODE REVIEW

File: typescript-operators-and-expressions/main.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const width: number = 10;
+const height: number = 10;
+const area: number = width * height;
+
+console.log('value of area:', area);
+console.log('type of area:', typeof area);
+
+const bill: number = 120;
+const payment: number = 140;
+const change: number = bill - payment;
```
Comment (shawnkost) [inline typescript-operators-and-expressions/main.ts:10]: `payment` & `bill` should be swapped. Hopefully you pay enough to cover your bill 😉

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looks great Blake! Just one small change here

File: typescript-operators-and-expressions/main.ts (hunk 1 above)
Comment (BlakexLim) [reply typescript-operators-and-expressions/main.ts:10]: hope this worked

File: typescript-operators-and-expressions/main.ts [hunk 2]
```diff
@@ -5,8 +5,8 @@ const area: number = width * height;
 console.log('value of area:', area);
 console.log('type of area:', typeof area);
 
-const bill: number = 120;
-const payment: number = 140;
+const bill: number = 140;
+const payment: number = 135;
 const change: number = bill - payment;
```
Comment (shawnkost) [inline typescript-operators-and-expressions/main.ts:10]: Not quite! We still want `payment` to be greater than `bill`, we just need to do `payment - bill` instead for `change`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small thing here!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**